Proposal-Process.md

```diff
@@ -0,0 +1,14 @@
+For a proposal to create or change a process foundation-wide, a PR should be
```

Comment:
"foundation-wide" is misleading. The TSC does not establish policies for the Foundation, which is a legal entity with it's own governing board of directors. The TSC establishes policies for the technical governance of the Top Level Projects that exist under the Foundation, yes, but that's not the same as "foundation wide".